templates/Web/Pages/Angular.List/src/app/app-shell/Param_SourceName_Kebab/list-form/list-form.component.css

```diff
@@ -0,0 +1,4 @@
+.col-11{
+    padding-left: 0px;
+    padding-right: 0px;
```

Comment:
surprised this actaully worked lol nice

Reply:
![image](https://user-images.githubusercontent.com/9086862/59814881-a8d41b00-92cb-11e9-8211-ae66d88bbb6f.png)